registry/configuring-registry-operator.adoc

```diff
@@ -0,0 +1,29 @@
+[id=configuring-registry-operator’]
+= Configuring the Image Registry Operator in {product-title}
+include::modules/common-attributes.adoc[]
+:context: configuring-registry-operator
+toc::[]
+
+The Image Registry Operator manages a single instance of the {product-title}
+registry. It manages all configuration of the registry including setting up
+registry storage.
+
+On initial startup, the Operator will create a default `image-registry` resource
+instance based on configuration detected in the cluster (e.g., what cloud storage
+type to use based on the cloud provider).
```

Comment:
It is better to highlight some configures are needed during installation for UPI on baremetal and vsphere, since there is no default storage configure for them, it will return: Unable to apply resources: storage backend not configured.
In this case, when try to describe clusteropeartor image-registry:
Status:
  Conditions:
    Last Transition Time:  2019-04-15T05:28:10Z
    Message:               The registry is ready
    Reason:                Ready
    Status:                True
    Type:                  Available
    Last Transition Time:  2019-04-16T05:42:07Z
    Message:               Unable to apply resources: storage backend not configured
    Reason:                Error
    Status:                True
    Type:                  Progressing
    Last Transition Time:  2019-04-16T05:42:07Z
    Message:               storage backend not configured
    Reason:                StorageNotConfigured
    Status:                True
    Type:                  Failing

Reply:
FYI https://github.com/openshift/installer/blob/master/docs/user/vsphere/install_upi.md#updating-image-registry-to-emptydir-storage-backend

Reply:
Thanks, @wzheng1! I added the following note, and am still working on the registry storage PR to go in the "Installing" book - https://github.com/openshift/openshift-docs/pull/14390.

"Storage is only automatically configured when you install on AWS."